app/src/main/java/com/infomaniak/drive/data/cache/DriveInfosController.kt

```diff
@@ -129,4 +141,40 @@ object DriveInfosController {
 
         return teamList.filter { drive.teams.account.contains(it.id) }
     }
-}
\ No newline at end of file
+
+    fun getCategories(fileCategoriesIds: Array<Int>? = null): List<Category> {
```

Comment:
This method only return categories for the currentDrive, i think it'll be good to rename this, example `getCurrentDriveCategories` what do you think ?